site-src/getting-started.md

```diff
@@ -1,61 +1,62 @@
 # Getting started with Gateway APIs
 
-## Installing CRDs
+The Gateway API provides a collection of Custom Resource Definitions (CRDs)
+that can be installed into any Kubernetes (>= 1.16) cluster. Most
+implementations install the CRDs for you, but they can also be manually
+installed. To get hands-on with the Gateway API, try the following resources:
 
-This project provides a collection of Custom Resource Definitions (CRDs) that can
-be installed into any Kubernetes (>= 1.16) cluster.
+**1.   [Install the Gateway API CRDs](#installing-gateway-api-crds)**
 
-To install the CRDs, please execute:
+_OR_
 
-```
-kubectl kustomize "github.com/kubernetes-sigs/gateway-api/config/crd?ref=v0.2.0" \
-| kubectl apply -f -
-```
-
-## Install an implementation
-
-[Multiple projects](implementations.md) implement the APIs defined by this
-project.  You will need to either install an implementation or verify that one
-is already setup for your cluster.
+**1.   [Install a Gateway controller](#installing-a-gateway-controller) from the 
+available implementations.**
 
-## Sample Gateway
+_THEN_
 
-Once you have the CRDs and an implementation installed, you are ready to
-use Gateway API.
+**2.   Try out one of the available guides:**
 
-In this example, we are installing three resources:
+- [Simple Gateway](simple-gateway.md) (a good one to start out with)
+- [HTTP routing](http-routing.md)
+- [HTTP traffic splitting](traffic-splitting.md)
+- [Routing across Namespaces](multiple-ns.md)
+- [Configuring TLS](tls.md)
+- [TCP routing](tcp.md)
 
-- An `acme-lb` GatewayClass which is being managed by a `acme.io/gateway-controller`
-  controller running in the cluster. Typically, a GatewayClass is provided by
-  the implementation and must be installed in the cluster.
-- A Gateway which is of type `acme-lb`:
-    - This gateway has a single HTTP listener on port 80 which selects HTTPRoutes
-      from all namespaces which have the label `app: foo` on them.
+After you're done, you can clean up after yourself by [uninstalling the 
+Gateway API CRDs](#uninstalling-the-crds)
 
-- Finally, we have an HTTPRoute resource which is attached to the above Gateway
-  and has two rules:
-    - All requests with path beginning with `/bar` are forwarded to my-service1
-      Kubernetes Service.
-    - All requests with path beginning with `/some/thing` AND have an HTTP header
-      `magic: foo` are forwarded to my-service2 Kubernetes Service.
+## Installing Gateway API CRDs
 
-With this configuration, you now have a Gateway resource which is forwarding
-traffic to two Kubernetes Services based on HTTP request metadata.
+The following command will install the Gateway API CRDs. This includes the
+GatewayClass, Gateway, HTTPRoute, TCPRoute, UDPRoute, and BackendPolicy. Note
```

Comment:
This may not need to be an exhaustive list, and it will likely grow out of date, but we also have TLSRoute in the current API.